### PR TITLE
SASS-1033 - Creation of 'name of your employer' page

### PR DESCRIPTION
**Pull request by ina-tu**
### Description
created the 'name of employer' page

Add a link to the relevant story in Jira
[SASS-857](https://jira.tools.tax.service.gov.uk/browse/SASS-857)

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?
- [x]  Have you checked to ensure its been rebased against the current version of master?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [x]  Have you checked for merge conflicts or any changes in the current master that may affect the current pull request? i.e. does it need another rebase?
- [x]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the journey tests? (where applicable)
- [x]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of master?
- [x]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [x]  Have you checked the PR Builder passes?

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -141,6 +141,13 @@ employment.employmentDetails.hiddenText.paymentNotOnYourP60.agent = if your clie
 employment.employmentDetails.hiddenText.tipsAndOtherPayments.individual = the amount of payments that were not on your P60
 employment.employmentDetails.hiddenText.tipsAndOtherPayments.agent = the amount of payments that were not on your client’s P60
 
+employment.employerName.heading.individual = What’s the name of your employer?
+employment.employerName.heading.agent = What’s the name of your client’s employer?
+employment.employerName.error.noEntry.individual = Enter the name of your employer
+employment.employerName.error.noEntry.agent = Enter the name of your client’s employer
+employment.employerName.error.name.limit = The employer name must be 74 characters or fewer
+employment.employerName.error.name.duplicate = You cannot add 2 employers with the same name
```
Comment (BenCosfordHMRC) [inline conf/messages:149]: duplicate message can be removed now

File: conf/messages.cy [hunk 2]
```diff
@@ -141,6 +141,13 @@ employment.employmentDetails.hiddenText.paymentNotOnYourP60.agent = if your clie
 employment.employmentDetails.hiddenText.tipsAndOtherPayments.individual = the amount of payments that were not on your P60
 employment.employmentDetails.hiddenText.tipsAndOtherPayments.agent = the amount of payments that were not on your client’s P60
 
+employment.employerName.heading.individual = What’s the name of your employer?
+employment.employerName.heading.agent = What’s the name of your client’s employer?
+employment.employerName.error.noEntry.individual = Enter the name of your employer
+employment.employerName.error.noEntry.agent = Enter the name of your client’s employer
+employment.employerName.error.name.limit = The employer name must be 74 characters or fewer
+employment.employerName.error.name.duplicate = You cannot add 2 employers with the same name
```
Comment (BenCosfordHMRC) [inline conf/messages.cy:149]: as above

File: it/controllers/employment/EmployerNameControllerISpec.scala [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+
+            errorSummaryCheck(expectedErrorCharLimit, Selectors.inputFieldSelector)
+            errorAboveElementCheck(expectedErrorCharLimit)
+          }
+        }
+      }
+    }
+  }
+
+  ".submit" should {
```
Comment (BenCosfordHMRC) [inline it/controllers/employment/EmployerNameControllerISpec.scala:282]: there's two ".submit should" cases can it be combined into one

**Review by BenCosfordHMRC — CHANGES_REQUESTED** [review]

File: it/controllers/employment/EmployerNameControllerISpec.scala (hunk 3 above)
Comment (ina-tu) [reply it/controllers/employment/EmployerNameControllerISpec.scala:206]: sure

File: conf/messages.cy (hunk 2 above)
Comment (ina-tu) [reply conf/messages.cy]: done

File: conf/messages (hunk 1 above)
Comment (ina-tu) [reply conf/messages]: done

**Review by BenCosfordHMRC — APPROVED** [review]

**Pull request merged**